dir.traversal.targets

```diff
@@ -86,6 +86,32 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
+  <Target Name="ProducesPackageId"
+          Returns="@(PackageIds)" 
```

Comment:
I understand that we will interact with the APIs via the run tool but what isn't clear is who will ever call `run produces` and what will they do with that output?